react/components/Tabs/Tab.js

```diff
@@ -28,7 +28,9 @@ class Tab extends Component {
           fullWidth ? 'w-100' : ''
         } ${tabStyle}
         v-mid relative h-regular ph6 t-body bg-transparent outline-0
-        `}>
+        ${className}`}
```

Comment:
custom `classNames` should not be allowed.

Reply:
Why? Since `className` is not allowed can I create a new prop to handle a local class?

Reply:
This `Tab` component can be accessed by the user, right? if you allow him/her to pass any className it will result in a possible (almost certain) consistency breakup.

Reply:
You can create a handle though. Ex: `vtex__tab`.

Reply:
Updated.